massdriver.yaml

```diff
@@ -190,12 +92,8 @@ params:
           title: Alarm Mode
           description: Enable and customize Function App metric alarms.
           type: string
-          default: AUTOMATED
+          default: DISABLED
```

Comment:
I don't mean removing the `mode`, as that would break the schema. We could use the `ui:hidden` flag to just hide it from the user for now.

Reply:
I believe it was both.

![image](https://user-images.githubusercontent.com/651833/214105237-b5298923-aa90-4fbc-b0d6-6a3482627dce.png)

It was a longgggg week and I didn't have the time to dig in. But! I do now. Since these are _latency_ metrics, I'm going to see if bumping them up will help. I ~think they are currently set at 500ms. I'll deploy to my personal org and let ya know.

Reply:
I hit Bloby with apache bench and the latency metrics are way below 500ms so I'm leaving it be!

<img width="1173" alt="Screenshot 2023-01-23 at 12 20 21 PM" src="https://user-images.githubusercontent.com/651833/214141575-4f9f47c0-07d2-45ec-a101-2ded511130f9.png">